src/map/battle.cpp

```diff
@@ -4033,14 +4042,14 @@ static int battle_calc_attack_skill_ratio(struct Damage* wd, struct block_list *
 			break;
 		case SR_EARTHSHAKER:
 			if (tsc && ((tsc->option&(OPTION_HIDE|OPTION_CLOAK|OPTION_CHASEWALK)) || tsc->data[SC_CAMOUFLAGE] || tsc->data[SC_STEALTHFIELD] || tsc->data[SC__SHADOWFORM])) {
-				//[(Skill Level x 150) x (Caster Base Level / 100) + (Caster INT x 3)] %
-				skillratio += -100 + 150 * skill_lv;
+				//[(Skill Level x 300) x (Caster Base Level / 100) + (Caster STR x 3)] %
```

Comment:
from this http://ro.gnjoy.com/news/update/View.asp?seq=235&curpage=1  and this https://www.divine-pride.net/database/skill/2328/

It seem the new base damage formula is
-> Normal target  -> -100 + 300 * skill_lv

-> Hidden target -> -100 + 400* skill_lv